cluster/ci/origin/images/release-base/log-pods.sh

```diff
@@ -48,7 +49,11 @@ while [[ "${finished}" == "no" ]]; do
 		fi
 		totalCount=$(( totalCount + 1 ))
 		completed_pods+=("${podname}")
-		oc log "${podname}"
+		if [[ -n "${container_name}" ]]; then
```

Comment:
You have to check `-n` here but you just assign to `${2}` -- will the assignment fail when there are not 2+ args?

Reply:
you're right, it will fail... let me add a default